### PR TITLE
Button style tweaks

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-351

#### Description
Update button styles based on new decisions made by Max B and Chris. Background colors will be correct once we start to use `swarm-constants` instead of `meetup-swatches`.

#### Screenshots (if applicable)
screen shot 2017-07-28 at 4 39 20 pm

### CODE REVIEW

File: assets/scss/components/_button.scss [hunk 1]
```diff
@@ -294,7 +301,7 @@ Class                   | Description
 		$textColor: $C_textPrimary
 	);
 	border-radius: 0;
-	font-weight: normal;
+	// font-weight: normal;
```
Comment (nlaz) [inline assets/scss/components/_button.scss]: Kill this comment? 🔪

Comment (mperrotti) [inline assets/scss/components/_button.scss]: The deed is done 🔪

File: assets/scss/components/_button.scss [hunk 2]
```diff
@@ -30,6 +30,14 @@ _(The `extend` directive can cause ugly cascade problems)_
 @include buttonBase();
 ```
 */
+
+//
+// Get these vars updated in sasstools before merging
```
Comment (mmcgahan) [inline assets/scss/components/_button.scss]: You can add the `Prelim` label and leave a note in the PR to help make sure this isn't merged prematurely

Comment (mperrotti) [inline assets/scss/components/_button.scss]: Good catch - thanks Mike

Comment (mperrotti) [inline assets/scss/components/_button.scss]: Just checked - they're in there. Removing from here...

**Pull request merged**